src/main/java/com/amaze/filemanager/fragments/MainFragment.java

```diff
@@ -1686,7 +1693,7 @@ protected Void doInBackground(Void... params) {
             public void onPostExecute(Void c) {
                 createViews(getLayoutElements(), true, (CURRENT_PATH), openMode, true, !IS_LIST);
                 pathname.setText(MAIN_ACTIVITY.getString(R.string.empty));
-                mFullPath.setText(MAIN_ACTIVITY.getString(R.string.searchresults));
+                mFullPath.setText(MAIN_ACTIVITY.getString(R.string.searchresults)+" "+query);
```

Comment:
You should have used [resource string formatting](https://code.tutsplus.com/tutorials/android-sdk-quick-tip-formatting-resource-strings--mobile-1775).